plugins/poll/plugin.rb

```diff
@@ -398,6 +398,18 @@ def vote
       end
     end
 
+    def remove_vote
+      post_id   = params.require(:post_id)
+      poll_name = params.require(:poll_name)
+
+      begin
+        poll, options = DiscoursePoll::Poll.vote(post_id, poll_name, [], current_user, remove: true)
```

Comment:
I'm personally not a huge fan of extending the `DiscoursePoll::Poll.vote` method to be able to remove vote because the original intend of the method is to cast a vote. A new method like `DiscoursePoll::Poll.remove_vote` would make it much clearer IMO.